src/components/CommentCard/CommentCard.module.scss

```diff
@@ -48,3 +48,83 @@
 		}
 	}
 }
+
+.productPageCommentCard * {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+}
+
+.productPageCommentCard {
+	display: flex;
+	flex-direction: column;
+	gap: 10px;
+	width: 600px;
+	height: 200px;
+	border: 2px solid $secondary-color;
+	border-radius: 20px;
+	padding: 20px !important;
```

Comment:
is the use of `!important` necessary? can you achieve this with better specificity application to your stlying? If yes, always go with that and only use the `!important` tag as a last-resort.

Reply:
I tried different ways but the component is wrapped around a parent component which overwrites this rule, so I used `!important` finallyç